TCAT/Controllers/ActionOnboardViewController.swift

```diff
@@ -119,7 +119,11 @@ class ActionOnboardViewController: UIViewController, CLLocationManagerDelegate {
             make.width.equalTo(image.snp.height)
             make.leading.equalToSuperview().offset(32)
             make.trailing.equalToSuperview().offset(-32)
-            make.top.equalToSuperview().offset(32)
+            if UIDevice().userInterfaceIdiom == .phone && UIScreen.main.nativeBounds.height == 2436 {
```

Comment:
I would name this boolean so its clear what condition your checking. Like `let isiPhoneXDevice: Bool = UIDevice().userInterfaceIdiom == .phone && UIScreen.main.nativeBounds.height == 2436`